src/js/lint-all.js

```diff
@@ -0,0 +1,309 @@
+                            /*
+                                Allow ES5 multi-line strings.
+                            */
+                            "no-multi-str": "off",
+                            "trailing-comma": "off"
+                        }
+                    }
+                }
+            },
+            "md": {
+                "enabled": true,
+                "includes": "{that}.options.config.sources.md",
+                "excludes": [],
+                "options": {
+                    "resolvePluginsRelativeTo": "@expand:fluid.module.resolvePath(%fluid-lint-all)",
+                    "overrideConfig": {
+                        "env": {
+                            "browser": true
+                        },
+                        "rules": {
+                            "no-undef": "off",
+                            "strict": "off",
+                            "no-unused-vars": "off",
+                            "no-console": "off"
+                        },
+                        "plugins": [
+                            "markdown"
+                        ]
+                    }
+                }
+            }
+        },
+        "json5lint": {
+            "enabled": true,
+            "includes": "{that}.options.config.sources.json5",
+            "excludes": []
+        },
+        "jsonlint": {
+            "enabled": true,
+            "includes": "{that}.options.config.sources.json",
+            "excludes": ["./package-lock.json"]
+        },
+        "lintspaces": {
+            "enabled": true,
+            "jsonindentation": {
+                "enabled": true,
+                "includes": "@expand:fluid.flatten({that}.options.config.sources.json, {that}.options.config.sources.json5)",
+                "excludes": ["./package-lock.json"],
+                "options": {
+                    indentation: "spaces",
+                    spaces: 4
+                }
+            },
+            "newlines": {
+                "enabled": true,
+                "includes": ["./src/**/*", "./tests/**/*", "./*"],
+                "excludes": ["./package-lock.json"],
+                options: {
+                    newline: true
+                }
+            }
+        },
+        "markdownlint": {
+            "enabled": true,
+            "includes": "{that}.options.config.sources.md",
+            "excludes": [],
+            options: { config: "@expand:fluid.require(%markdownlint-config-fluid/.markdownlintrc.json)" }
+        },
+        "mdjsonlint": {
+            "enabled": true,
+            "includes": "{that}.options.config.sources.md",
+            "excludes": []
+        },
+        "stylelint": {
+            "enabled": true,
+            "includes": "@expand:fluid.flatten({that}.options.config.sources.css, {that}.options.config.sources.scss)",
+            "excludes": [],
+            options: {
+                configFile: "%fluid-lint-all/.stylelintrc.json"
```

Comment:
I'm getting an error related to this file when running the lint script with default configuration:

```bash
(node:19524) UnhandledPromiseRejectionWarning: Error: ENOENT: no such file or directory, open '/Users/ned/Sites/fluidic-11ty/%fluid-lint-all/.stylelintrc.json'
```

I do have a `.stylelintrc.json` file in the project where I'm testing this (see here: https://github.com/fluid-project/fluidic-11ty/blob/main/.stylelintrc.json

Reply:
Thanks for working to help review this, I appreciate the additional set of eyes and help exercising things, especially in downstream projects.

How are you actually running the command?  Can you stash you work in a branch where I can try it out?

Reply:
@the-t-in-rtf I'll push to a branch where you can see this behaviour.

Reply:
If you clone this branch: https://github.com/greatislander/fluidic-11ty/tree/stylelint-error

And run `npm run lint`, you should see the error. In this commit (https://github.com/greatislander/fluidic-11ty/commit/1ec1a1a6d120c131636374b0036b15250329be10) I removed the custom fluid-lint-all configuration specifying a custom stylelint configuration file which I used to work around the issue.

Reply:
OK, I found a bug that prevented resolving the path to the default file, and fixed that.   @greatislander, I cut a dev release that should work downstream without a .stylelinrc.json: `1.0.0-dev.20210111T144240Z.ee70ce3.GH-1`

Reply:
Thanks, @the-t-in-rtf! I can confirm that this fixes the issue for me.